Lib/gftools/builder/operations/fontc/__init__.py

```diff
@@ -0,0 +1,61 @@
+from pathlib import Path
+from typing import List
+from gftools.builder.operations import OperationBase
+
+_FONTC_PATH = None
+
+
+# should only be called once, from main, before doing anything else. This is a
+# relatively non-invasive way to smuggle this value into FontcOperationBase
+def set_global_fontc_path(path: Path):
+    global _FONTC_PATH
+    assert _FONTC_PATH is None, "set_global_fontc_path should only be called once"
+    _FONTC_PATH = path
+
+
+class FontcOperationBase(OperationBase):
+    @property
+    def variables(self):
+        vars = super().variables
+        vars["fontc_path"] = _FONTC_PATH
+        args = vars.get("args")
+        if args:
+            vars["args"] = rewrite_fontmake_args_for_fontc(args)
+
+        return vars
+
+
+def rewrite_fontmake_args_for_fontc(args: str) -> str:
+    out_args = []
+    arg_list = args.split()
+    # reverse so we can pop in order
```

Comment:
Nit. I'm probably being dense here but couldn't skip the reverse part and just keep popping the first element by using `args.pop(0)`,